src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java

```diff
@@ -95,49 +108,52 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
             }
         }
 
-        final Colony colony = ColonyManager.getColony(colonyId);
+        String playerName = null;
+        if (args.length >= 2)
+        {
+            playerName = args[1];
+        }
 
+        final Colony colony = ColonyManager.getColony(colonyId);
         if (colony == null)
         {
             sender.sendMessage(new TextComponentString(String.format(COLONY_NULL, colonyId)));
             return;
         }
 
-        String playerName = null;
-        if (args.length >= 2)
+        if (playerName == null || playerName.isEmpty())
         {
-            playerName = args[1];
+            sender.sendMessage(new TextComponentString(NO_PLAYER));
+            return;
         }
 
-        if (playerName == null || playerName.isEmpty())
+        final EntityPlayer player = sender.getEntityWorld().getPlayerEntityByName(playerName);
+
+        executeShared(server, sender, colony, player);
+    }
+
+    private void executeShared(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, final Colony colony, final EntityPlayer player) throws CommandException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "server". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)